plugins/woocommerce/src/Internal/Admin/Logging/FileV2/File.php

```diff
@@ -430,6 +430,10 @@ protected function create(): bool {
 	 * @return bool
 	 */
 	public function write( string $text ): bool {
+		if ( $text === '' ) {
```

Comment:
```suggestion
		if ( '' === $text ) {
```

Our linter, unfortunately, requires [yoda conditions](https://en.wikipedia.org/wiki/Yoda_conditions).